README.md

```diff
@@ -1,35 +1,33 @@
 # The SpatialOS GDK for Unity (alpha)
 
-The SpatialOS Game Development Kit (GDK) for Unity enables game developers to build multiplayer [Unity](http://unity3d.com) games with [SpatialOS](https://docs.improbable.io/reference/latest/shared/concepts/spatialos). It will be made up of three parts:
+The SpatialOS Game Development Kit (GDK) for Unity allows you to quickly and easily build and host Unity multiplayer games. These games can use multiple server-side game engines across one seamless world to create new kinds of gameplay.
 
-* the GDK Core, a performant, data-driven [SpatialOS](https://docs.improbable.io/reference/latest/shared/concepts/spatialos) multiplayer integration.
+The GDK is composed of three layers:
 
-* a library of solutions for hard or common networked game development problems via Feature Modules.
+**The GDK Core:** a performant, data-oriented integration with our cloud platform SpatialOS, based on the familiar Unity-native workflows.
 
-* the example game, which both tests and demonstrates the Feature Modules; for use as a starting point or resource for game development.
+**The GDK Feature Modules:** a library of solutions for hard or common networked game development problems, such as Character Movement and Shooting.
 
-This is an alpha release of the GDK Core which is the foundation of the product. To find out more see the [GDK blogpost](https://improbable.io/games/blog/unity-gdk-our-first-steps).
+**The First Person Shooter (FPS) Starter Project:** a starting sandbox project for the GDK that enables you and your friends to experience the true scale of SpatialOS, providing a solid foundation for entirely new games.
+
+[Get started](https://docs.improbable.io/unity/alpha/get-started) with the SpatialOS GDK for Unity, by setting up the FPS Starter Project.
 
 ---
 
-* [Setup Guide](https://docs.improbable.io/unity/alpha/setup-and-installing#set-up-your-machine)
+* [Get started](https://docs.improbable.io/unity/alpha/get-started)
 * [Documentation](https://docs.improbable.io/unity/alpha/)
-* [Development Roadmap](https://trello.com/b/29tMKyQC)
-* [Discord](https://discord.gg/SCZTCYm)
-* [Forums](https://forums.improbable.io/latest?tags=unity-gdk)
-* [Github Issues](https://github.com/spatialos/UnityGDK/issues)
-* [Known Issues](https://docs.improbable.io/unity/alpha/known-issues)
-* [Sign up for e-mail updates](http://go.pardot.com/l/169082/2018-06-25/27mhsb)
 
----
+* [Development roadmap](https://trello.com/b/29tMKyQC)
+* [Issue reporting](https://github.com/spatialos/UnityGDK/issues) ([Known issues](https://docs.improbable.io/unity/alpha/known-issues))
+* [Contributions](/.github/CONTRIBUTING.md)
 
-## Recommended use
+* Community: [Discord](https://discord.gg/SCZTCYm) - [Forums](https://forums.improbable.io/latest?tags=unity-gdk) - [Mailing list](http://go.pardot.com/l/169082/2018-06-25/27mhsb)
```

Comment:
do we want this in one line? or as a list?

Reply:
ah actually it looks good. nvm

Reply:
Neater in a line, IMHO. Keep the readme short.